<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordingWizard</class>
 <widget class="QWizard" name="RecordingWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recording Wizard</string>
  </property>
  <widget class="QWizardPage" name="selectCostumeswizardPage">
   <property name="title">
    <string>Recording configuration</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="costumesLabel">
      <property name="text">
       <string>Costumes:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="sensorsLabel">
      <property name="text">
       <string>Sensors:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="costumesToRecordListWidget"/>
    </item>
    <item row="1" column="1">
     <widget class="QListWidget" name="sensorsToRecordListWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="outputWizardPage">
   <property name="title">
    <string>Data output</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>Output file:</string>
        </property>
        <property name="buddy">
         <cstring>outputLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="outputPushButton">
        <property name="text">
         <string>Choose</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>300</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>costumesToRecordListWidget</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>RecordingWizard</receiver>
   <slot>onCostumeSelectionChange(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputPushButton</sender>
   <signal>clicked()</signal>
   <receiver>RecordingWizard</receiver>
   <slot>onChoose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RecordingWizard</receiver>
   <slot>onOutputFilePathEdit(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RecordingWizard</sender>
   <signal>currentIdChanged(int)</signal>
   <receiver>RecordingWizard</receiver>
   <slot>pageChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sensorsToRecordListWidget</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>RecordingWizard</receiver>
   <slot>onSensorSelectionChanged(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costumesToRecordListWidget</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>RecordingWizard</receiver>
   <slot>onCostumeChange(QListWidgetItem*,QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onCostumeSelectionChange(QListWidgetItem*)</slot>
  <slot>onChoose()</slot>
  <slot>onOutputFilePathEdit(QString)</slot>
  <slot>pageChanged(int)</slot>
  <slot>onSensorSelectionChanged(QListWidgetItem*)</slot>
  <slot>onCostumeChange(QListWidgetItem*,QListWidgetItem*)</slot>
 </slots>
</ui>
